lib/collection.js

```diff
@@ -242,6 +245,13 @@ Object.defineProperty(Collection.prototype, 'hint', {
   }
 });
 
+Object.defineProperty(Collection.prototype, 'ns', {
```

Comment:
In `Cursor`, this is defined as [namespace](https://github.com/mongodb/node-mongodb-native/blob/master/lib/cursor.js#L1074). Can we keep these consistent?